test/deploy/rackhd_redfish_service_validator.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# This is a helper script to run the Redfish Service Validator on RackHD
+# Usage:
+# ./rackhd_redfish_service_validator.sh host:port
+
+if [ -z $1 ]; then
```

Comment:
Please add a -h option to catch and display usage.
I tried -h and it started installing.